src/modules/app/route-configs/ExtrasRouteConfig.js

```diff
@@ -2,7 +2,7 @@
 
 import { RouteConfig } from './RouteConfig'
 import { extrasEndpoint, Payload, ExtraModel } from '@integreat-app/integreat-api-client'
-import type { Dispatch, GetState, Route } from 'redux-first-router'
+import type { Dispatch, GetState, Route } from 'redux-first-router/dist/flow-types'
```

Comment:
```suggestion
import type { Route } from 'redux-first-router'
```
We don't need to import Dispatch and GetState, since flow knows their type by the Route type